jobs/build/ocp/Jenkinsfile

```diff
@@ -401,8 +424,85 @@ node(TARGET_NODE) {
+                  def build_result = sh (
+                    script: "ose_images.sh --user ocp-build build_container --branch rhaos-${BUILD_VERSION}-rhel-7 --workingdir ${pwd()} --group base --repo ${PUDDLE_REPO}"
+                    returnStatus: true
+                  ) > 0
+                  def failed_list_file = "${pwd()}/logs/failed_image_list"
+                  IMG_FAILURES = []
+                  if(fileExists(failed_list_file)) {
+                    def failed_image_list = readFile(failed_list_file)
+                    IMG_FAILURES = failed_image_list.split("\\r?\\n");
+                  }
+
+                  def success_list_file = "${pwd()}/logs/success_image_list"
+                  IMG_SUCCESSES = []
+                  if(fileExists(success_list_file)){
+                    def success_image_list = readFile(success_list_file)
+                    IMG_SUCCESSES = success_image_list.split("\\r?\\n");
+                  }
+
+                  required_images = [
+                    "openshift-enterprise-apb-base-docker",
+                    "openshift-enterprise-asb-docker",
+                    "openshift-enterprise-base-docker",
+                    "openshift-enterprise-cluster-capacity-docker",
+                    "openshift-enterprise-deployer-docker",
+                    "openshift-enterprise-docker",
+                    "openshift-enterprise-docker-builder-docker",
+                    "openshift-enterprise-dockerregistry-docker",
+                    "openshift-enterprise-egress-router-docker",
+                    "openshift-enterprise-federation-docker",
+                    "openshift-enterprise-haproxy-router-docker",
+                    "openshift-enterprise-keepalived-ipfailover-docker",
+                    "openshift-enterprise-mediawiki",
+                    "openshift-enterprise-mediawiki-docker",
+                    "openshift-enterprise-node-docker",
+                    "openshift-e",
+                    "openvswitch-docker",
+                    "openshift-enterprise-pod-docker",
+                    "openshift-enterprise-postgresql",
+                    "openshift-enterprise-recycler-docker",
+                    "openshift-enterprise-service-catalog-docker",
+                    "openshift-enterprise-sti-builder-docker"
+                  ]
+
+                  if( build_result != 0){
+
+                    if(IMG_FAILURES.size() == 0 && IMG_SUCCESSES.size() == 0){
+                      error("ose_images.sh threw a fatal error while building images.")
+                    }
+                    if(IMG_FAILURES.size() > 0){
+                      fails = []
+                      for(String img: IMG_FAILURES){
+                        if(required_images.contains(img)){
+                          fails.add(img)
+                        }
+                      }
+                      if(fails.size() > 0){
+                        msg = "One or more required images failed to build.\n"
```

Comment:
Right, my comment was just that you have the full list of failures but are only exposing the ones that are required -- that may be annoying for someone debugging a failure